snippets/formatNumber.md

```diff
@@ -0,0 +1,32 @@
+---
+title: formatNumber
+tags: date,math,string,intermediate
+---
+
+Returns a number in readable format.
+
+Use `Number.prototype.toString()` to convert a number to string.
+Use `for` loop to loop through the characters in string. Write to a new string variable one character at a time, adding a comma at every third character. Return the newly created string.
+
+
+```js
+const formatNumber = number => {
+  let numberAsString = number.toString();
+  let formatted = '';
+  let count = 0;
+  for (let i = numberAsString.length - 1; i >= 0; i--) {
+    formatted = numberAsString[i] + formatted;
+    count++;
+    // add a comma to every three numbers
+    if (count % 3 == 0 && i != 0) {
+      formatted = ',' + formatted;
+    }
+  }
+  return formatted;
+}
+```
+
+```js
+formatNumber(123456); // '123,456'
```

Comment:
66067304

Reply:
Do you mean, should I add that to examples?

Reply:
The two examples are more than enough to explain the snippet. No need to add any more.